### PR TITLE
DOC-4130 added warning about nm-cloud-setup.service to installer page

### PR DESCRIPTION
**Pull request by andy-stark-redis**
[DOC-4130](https://redislabs.atlassian.net/browse/DOC-4130)

[DOC-4130]: https://redislabs.atlassian.net/browse/DOC-4130?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/integrate/redis-data-integration/ingest/installation/_index.md [hunk 1]
```diff
@@ -34,7 +34,78 @@ You must run the RDI installer as a privileged user because it installs
...
+
+```bash
+systemctl disable nm-cloud-setup.service nm-cloud-setup.timer
+reboot
+```
+
+See
+[Rancher support and maintenance terms](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-8-6/)
+for more information about the OS versions that have been tested with Rancher
+managed K3s clusters.
```
Comment (dwdougherty) [inline content/integrate/redis-data-integration/ingest/installation/_index.md:78]: Seems like is should be "...Rancher-managed..." in this context? Feel free to ignore.

**Review by dwdougherty — APPROVED** [review]
Just one wee comment that you may ignore. LGTM otherwise.

File: content/integrate/redis-data-integration/ingest/installation/_index.md (hunk 1 above)
Comment (andy-stark-redis) [reply content/integrate/redis-data-integration/ingest/installation/_index.md]: Well spotted! This is actually adapted from the K3s docs - I just copied and pasted this bit without reading it carefully enough.

**Comment by andy-stark-redis** [comment]
Thanks @dwdougherty !

**Pull request merged**